### ISSUE TITLE
NullPointerException while publish with mergeToOneReport

### ISSUE THREAD

**Issue by ViceIce** [jenkinsci/code-coverage-api-plugin#100]
I've trying the new `mergeToOneReport` merge feature but getting 'java.lang.NullPointerException'.

**config:**
```groovy
publishCoverage adapters: [istanbulCoberturaAdapter(mergeToOneReport: true, path: 'coverage/**/cobertura-coverage.xml')], sourceFileResolver: sourceFiles('STORE_LAST_BUILD'), tag : 'frontend'
```

**log:**
```
[2019-07-16T14:29:11.321Z] [Pipeline] publishCoverage
[2019-07-16T14:29:11.392Z] Publishing Coverage report....
[2019-07-16T14:29:12.138Z] A total of 1 reports were found
[2019-07-16T14:29:12.142Z] Source File Navigation is enabled - Current level: STORE_LAST_BUILD
[2019-07-16T14:29:12.143Z] 88 source files need to be copied
[2019-07-16T14:29:15.548Z] Error when executing always post condition:
[2019-07-16T14:29:15.549Z] java.lang.NullPointerException
[2019-07-16T14:29:15.549Z] 	at io.jenkins.plugins.coverage.CoverageProcessor.lambda$convertResultToAction$1(CoverageProcessor.java:139)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174)
[2019-07-16T14:29:15.549Z] 	at java.util.TreeMap$ValueSpliterator.tryAdvance(TreeMap.java:2921)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[2019-07-16T14:29:15.549Z] 	at java.util.stream.ReferencePipeline.findAny(ReferencePipeline.java:469)
[2019-07-16T14:29:15.549Z] 	at io.jenkins.plugins.coverage.CoverageProcessor.convertResultToAction(CoverageProcessor.java:140)
[2019-07-16T14:29:15.549Z] 	at io.jenkins.plugins.coverage.CoverageProcessor.performCoverageReport(CoverageProcessor.java:112)
[2019-07-16T14:29:15.549Z] 	at io.jenkins.plugins.coverage.CoveragePublisher.perform(CoveragePublisher.java:91)
[2019-07-16T14:29:15.549Z] 	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:80)
[2019-07-16T14:29:15.549Z] 	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:67)
[2019-07-16T14:29:15.549Z] 	at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
[2019-07-16T14:29:15.549Z] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[2019-07-16T14:29:15.549Z] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[2019-07-16T14:29:15.549Z] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[2019-07-16T14:29:15.549Z] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[2019-07-16T14:29:15.549Z] 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by ViceIce** [comment]
https://github.com/jenkinsci/code-coverage-api-plugin/blob/4367573b413261884bec8527c5ccf82b2e7cc5f7/src/main/java/io/jenkins/plugins/coverage/CoverageProcessor.java#L139

Maybe the tag gots lost while merging?

**Comment by cizezsy** [comment]
Fixed in version 1.0.13, thanks for the report~
